index.html

```diff
@@ -0,0 +1,2688 @@
+                        </p>
+                      </div>
+                    </article>
+                  </div>
+
+                  <div class="col-12 order-3">
+                    <article class="advertisement">
+                      <div class="category">SPONSORED INSIGHT</div>
+                      <div class="info_1">
+                        <h3>
+                          <a href="#"
+                            >What Can a Degree in Public Health Do For You?</a
+                          >
+                        </h3>
+                        <p>
+                          The field of public health is a powerful draw for
+                          those who feel inspired to make a positive difference
+                          in their communities. Find out more.
+                        </p>
+                      </div>
+                    </article>
+                  </div>
+
+                  <div class="col-12 order-4">
+                    <article class="advertisement">
+                      <div class="category">SPONSORED INSIGHT</div>
+                      <div class="info_1">
+                        <h3><a href="#">Wizard of Oz Special Edition</a></h3>
+                        <p>
+                          Celebrate the 80th anniversary of one of the most
+                          iconic American films with Newsweek's Special Edition.
+                        </p>
+                      </div>
+                    </article>
+                  </div>
+
+                  <div class="col-12 order-5">
+                    <article class="advertisement">
+                      <div class="category">SPONSORED INSIGHT</div>
+                      <div class="info_1">
+                        <h3><a href="#">Mister Rogers Special Edition</a></h3>
+                        <p>
+                          Commemorate one of the most beloved childeren's
+                          programming hosts.
+                        </p>
+                      </div>
+                    </article>
+                  </div>
+                </div>
+              </div>
+            </aside>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-12 col-md-3">
+            <div class="cover">
+              <div class="picture-frame">
+                <div class="picture-frame-image">
+                  <a href="#">
+                    <picture>
+                      <img
+                        class="img-fluid"
+                        src="images/nov_15_issue.webp"
+                        alt="Magazine Cover"
+                      />
+                    </picture>
+                  </a>
+                </div>
+                <div class="picture-frame-overlay">
+                  <a href="#">NOVEMBER 15 ISSUE</a>
+                </div>
+              </div>
+              <a class="see-all-features" href="#">SEE ALL FEATURES ></a>
+            </div>
+          </div>
```

Comment:
Please take a look a the source code of the original page to get a better idea on how to structure this.

Notice on the source code of the original page that this first article is in the same `row` than the remaining three. Take some time to explore this section using the devtools of your preferred browser, that might give you the best idea on how they are doing it so you can try to mimic the layout:

![image](https://user-images.githubusercontent.com/2121481/71727152-64dbda00-2e07-11ea-9642-d9d48fec47df.png)

Reply:
I am having trouble with this one.